src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -105,7 +97,19 @@ class ChaptersButton extends TextTrackButton {
       }));
     }
 
-    if (chaptersTrack) {
+    if (chaptersTrack && !chaptersTrack.cues) {
```

Comment:
This seems questionable for me when determining when to add event listeners.

`chaptersTrack.cues` === null (native, null on instantiation—0 after setup)
`chaptersTrack.cues` === object (emulated)